### ISSUE TITLE
Unable to reproduce the map accuracy of the yolo12n

### ISSUE THREAD

**Issue by zys1994** [sunsmarterjie/yolov12#102]
The map50-95 accuracy of the yolo12n I trained is 39.9. I trained with 8-V100, batchsize256(32/card), with albumentations. See the list below for details. Could you help me analyze how to reproduce it further? Sincerely hope to receive your suggestions！
```
task: detect
mode: train
model: {}/ultralytics/ultralytics/cfg/models/12/yolo12n_coco.yaml
data: {}/ultralytics/ultralytics/cfg/datasets/coco.yaml
epochs: 600
time: null
patience: 100
batch: 256
imgsz: 640
save: true
save_period: -1
cache: false
device:
- 0
- 1
- 2
- 3
- 4
- 5
- 6
- 7
workers: 16
project: null
name: yolo12n_coco_train_b512_8ka_lr001_mosaic
exist_ok: false
pretrained: true
optimizer: auto
verbose: true
seed: 0
deterministic: true
single_cls: false
rect: false
cos_lr: false
close_mosaic: 10
resume: false
amp: true
fraction: 1.0
profile: false
freeze: null
multi_scale: false
overlap_mask: true
mask_ratio: 4
dropout: 0.0
val: true
split: val
save_json: false
save_hybrid: false
conf: null
iou: 0.7
max_det: 300
half: false
dnn: false
plots: true
source: null
vid_stride: 1
stream_buffer: false
visualize: false
augment: false
agnostic_nms: false
classes: null
retina_masks: false
embed: null
show: false
save_frames: false
save_txt: false
save_conf: false
save_crop: false
show_labels: true
show_conf: true
show_boxes: true
line_width: null
format: torchscript
keras: false
optimize: false
int8: false
dynamic: false
simplify: true
opset: null
workspace: null
nms: false
lr0: 0.01
lrf: 0.01
momentum: 0.937
weight_decay: 0.0005
warmup_epochs: 3.0
warmup_momentum: 0.8
warmup_bias_lr: 0.1
box: 7.5
cls: 0.5
dfl: 1.5
pose: 12.0
kobj: 1.0
nbs: 64
hsv_h: 0.015
hsv_s: 0.7
hsv_v: 0.4
degrees: 0.0
translate: 0.1
scale: 0.5
shear: 0.0
perspective: 0.0
flipud: 0.0
fliplr: 0.5
bgr: 0.0
mosaic: 1.0
mixup: 0.0
copy_paste: 0.0
copy_paste_mode: flip
auto_augment: randaugment
erasing: 0.4
crop_fraction: 1.0
cfg: null
tracker: botsort.yaml
save_dir: {}
```

[results.csv](https://github.com/user-attachments/files/20075354/results.csv)

**Comment by sunsmarterjie** [comment]
1. Could you try to train again with 4 GPUs?    
2. V100 does not support FlashAttention, this does not align with my pre-training (even though theoretically consistent). I suggest to reproduce the results with FlashAttention.

**Comment by zys1994** [comment]
> 1. Could you try to train again with 4 GPUs?
> 2. V100 does not support FlashAttention, this does not align with my pre-training (even though theoretically consistent). I suggest to reproduce the results with FlashAttention.

I trained from scratch. Is a pre-trained model required to reproduce map95-40.4? If I use 4 cards to reproduce, should I set the batch size to 256 (64/card)?

**Comment by sunsmarterjie** [comment]
No pre-trained models are need. Yes, keep the batch size to 256.

**Comment by zys1994** [comment]
> No pre-trained models are needed. Yes, keep the batch size to 256. Refer to the log file during reproducing [log](https://github.com/sunsmarterjie/yolov12/blob/main/logs/yolov12_turbo_n.csv) (Note: The final saved model may exhibit slight fluctuations compared to the performance shown in the logs — you should verify the final performance using the saved model).

Thank you very much! Is this result from 4 card training or 8 card training?

**Comment by sunsmarterjie** [comment]
4
